packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -3485,7 +3485,10 @@ class _DropSliderValueIndicatorPathPainter {
       return;
     }
     assert(!sizeWithOverflow.isEmpty);
-
+    if (labelPainter.text == null) {
```

Comment:
This seems to only fix the issue for the `DropSliderValueIndicator` Shape. The error is still exist for other value indicator shape. It doesn't look like the fix should be made in all the shapes but in the `slider.dart` itself.  Skip drawing when label is null.

Reply:
You are right, I changed the implementation